index.html

```diff
@@ -120,12 +120,12 @@
         window.addEventListener(
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
-    <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
```

Comment:
I am actually not using SSI at all - I cannot process them in my webserver. I assumed, perhaps incorrectly, that making the reference relative to the file makes for a safer inclusion.